kubernetes/worker-ubuntu16-04.yaml

```diff
@@ -3,6 +3,9 @@ kind: Deployment
 metadata:
   name: worker-ubuntu16-04
   namespace: buildbarn
+  annotations:
```

Comment:
No service exists for the workers which makes sense so perhaps scraping the deployment is the best option?